adapter/ipListChecker/cfg.proto

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Google Inc.
```

Comment:
So why do we need to have these things expressed in protos? Seems awfully painful. Can't we just have 'em all defined in Go and use reflection?